### PR TITLE
Traceback is now collected from source

### PR DESCRIPTION
**Pull request by marceloFA**
Previously we collected the error traceback from a .txt file, which was temporary a temporary workaround.
This pull request implements ``get_traceback_from_sript`` which will now collect the error traceback directly from the standard error after the execution of the script in a subprocess.

Another small addition of this pull is a simple command-line argument parser to facilitate the development. Also, further in the future, this arg parser can be used to provide useful features through the cmd when pycee becomes a pip installable module (I dream of this every night).

Tests were updated accordingly. commits have messages and unformatted code was formatted (please ignore the last commit which was just formatting a ton of code).

### CODE REVIEW

**Comment by marceloFA** [comment]
All tests are passing but right now continuous integration is linting before testing and there are still linting faults to be resolved

**Comment by LeonardoFurtado** [comment]
> 
> 
> All tests are passing but right now continuous integration is linting before testing and there are still linting faults to be resolved

The error appears to be due to variables not defined in other files, can you solve this?
And take a look at #12

File: pycee/answers.py [hunk 1]
```diff
@@ -66,55 +65,56 @@ def get_questions(query):
...
         answer_body = response.json()["items"][0]["body"]
         answers_bodies.append(answer_body)
     return answers_bodies
 
 
-
-# Method to summary an answer body and remove html tags and formating
-
 def parse_summarizer(answer_body):
+    ''' Method to summary an answer body
```
Comment (LeonardoFurtado) [inline pycee/answers.py:97]: Change ' to "

**Review by LeonardoFurtado — COMMENTED** [review]
hahahahah

**Pull request merged**